### DISCUSSION TITLE
Add option to disable 'event emit' in the REST api

### DISCUSSION THREAD

**Discussion by arladmin** [directus/directus#16182] [Feature Requests]
Followed from this discussion: https://github.com/directus/directus/discussions/16181

**Request**: to have the option to disable event emitting when creating/updating/deleting via the REST api.

**Use-Case**: when manipulation of items is done externally, via the REST api, in response to create/update triggers raised by Webhook/Flow/Hook, a circular cascade of triggers gets set into motion which is undesirable.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
